frontend-react/src/pages/admin/AdminMain.tsx

```diff
@@ -0,0 +1,52 @@
+import { Suspense } from "react";
+import { Helmet } from "react-helmet";
+import { NetworkErrorBoundary } from "rest-hooks";
+
+import HipaaNotice from "../../components/HipaaNotice";
+import Spinner from "../../components/Spinner";
+import { useOrgName } from "../../utils/OrganizationUtils";
+import { ErrorPage } from "../error/ErrorPage";
+import { OrgSettingsTable } from "../../components/Admin/OrgSettingsTable";
+
+const OrgName = () => {
+    const orgName: string = useOrgName();
+    return (
+        <span id="orgName" className="text-normal text-base">
+            Admin settings for {orgName}
```

Comment:
Can drop {orgName} right since this page is for all ors?

Reply:
@sean-usds yes! fixed.